packages/web3-eth/test/integration/subscription_logs.test.ts

```diff
@@ -51,43 +51,42 @@ const makeFewTxToContract = async ({
 };
 describeIf(isWs)('subscription', () => {
 	let clientUrl: string;
-	let accounts: string[] = [];
 	let web3Eth: Web3Eth;
 	let providerWs: WebSocketProvider;
+	let contractInstance: Contract<typeof BasicAbi>;
 	let contract: Contract<typeof BasicAbi>;
 	let deployOptions: Record<string, unknown>;
 	let sendOptions: Record<string, unknown>;
 	let from: string;
```

Comment:
since this is only assigned inside on test, I suggest declaring it there.
```suggestion
```

Reply:
Actually, this variable was used inside both `beforeAll` and in the test case. Now, with the new code modifications, it is used inside only one code block.

Reply:
fixed